lib/model/store.dart

```diff
@@ -910,6 +910,24 @@ class PerAccountStore extends PerAccountStoreBase with ChangeNotifier, EmojiStor
     return _messages.sendMessage(destination: destination, content: content);
   }
 
+  @override
+  bool? getEditMessageErrorStatus(int messageId) {
+    assert(!_disposed);
+    return _messages.getEditMessageErrorStatus(messageId);
+  }
+
+  @override
+  void editMessage({required int messageId, required String content}) {
+    assert(!_disposed);
+    return _messages.editMessage(messageId: messageId, content: content);
```

Comment:
nit: let's move these up to sit next to the other MessageStore proxy implementations

(Should do the same with `sendMessage` too. I guess that should have happened when we moved `sendMessage` to MessageStore, 942aa87a52dc5b911c1f04b9ed774536e1ce1518 / #1455, oh well.)